capture_hdr_loop.py

```diff
@@ -1,68 +1,65 @@
-"""
-Capture HDR frames in a loop (while actively changing some HDR settings).
+"""Capture HDR frames in a loop (while actively changing some HDR settings).
+
+Three sets of settings are chosen. These settings provide a dynamic range of
+about 9-10 stops, with a decent low- and highlight frame which is sufficient
```

Comment:
comma after frame? To ensure we don't mean that the hightlight frame is sufficiont to grab data on most scenes but the HDR settings